product_docs/docs/pem/9/certificates/index.mdx

```diff
@@ -89,7 +89,7 @@ The PEM agent installed with the PEM server monitors the expiration date of the
 
 ## Generate a common agent certificate and key pair
 
-You can generate one agent certificate and key pair and use them for all the PEM agents.
+Generate one agent certificate and key pair and use them for all the PEM agents.
```

Comment:
```suggestion
By creating and using a single Postgres user for all PEM agents (rather than one user per agent as is the default) it is possible to use the same certificate for all agents. These instructions explain how to create a user, generate an agent certificate and key pair, and use them for all PEM agents.
```